Creating a roadmap for project team initiation (#67)
uwhackweek/hackweeks-as-a-service: docs/training/projects/project-during.md

* How do I work best?
* What skills / background am I bringing to this work?
* Why is this work important to me?
* What conditions do I need to thrive?
* How will I know I've had a successful hackweek?
```
Next, each person should take another 3 minutes to write some of this information on your card. Start with your name and preferred pronouns, if you wish. Then include other information from your reflections above. Get creative, and don't worry about capturing everything. Use words or sketches as you like!

Finally, give each member of your team up to 1 minute each (keep it short!) to share their name and a couple of items from their card. Be sure everyone has a chance to share. When you're done, keep your card in an accessible, readable location on the table so that others can look at it throughout the afternoon.

### Step 2: Group Norms and Process

Next, let's have a 10 - 15 minute conversation (depending on team size) about the group culture we hope to create during the week. 

```{admonition} GROUP PROCESS CHECK-IN
* How will we be making decisions?
* Who decides what tasks will be worked on?
* How will we navigate conflict?
* How will we recognize / attribute effort?
```

> (need some facilitation approach to ensure everyone is heard. 1-2-4 all?)

Reviewer: just flagging this as a TODO

### Step 3: Brainstorming Tasks

Now that the team has had a chance to get to know each other and talk about your process and systems, it is time to begin identifying tasks. 

```{admonition} CREATING A TASKBOARD

> * List the tasks that will need to be done (starting with any that the project leader may have already identified)
> * All team members are encouraged to contribute here. All ideas and perspectives are needed.
> * Identify any clusters of common tasks or tasks that must be completed sequentially
> * Team members volunteer for tasks that match their personal goals and interests, identify opportunities for working in pairs or small groups
> * Identify if any tasks require skills or knowledge that the current team does not yet have (the project team helper may be able to connect you with someone to provide those skills or knowledge)

Reviewer 2: ```suggestion
All team members are encouraged to contribute here. All ideas and perspectives are needed.
* List the tasks that will need to be done (starting with any that the project leader may have already identified).
* Identify any clusters of common tasks or tasks that must be completed sequentially.
* Team members volunteer for tasks that match their personal goals and interests, identify opportunities for working in pairs or small groups.
* Identify if any tasks require skills or knowledge that the current team does not yet have (the project team helper may be able to connect you with someone to provide those skills or knowledge).
```

Reviewer 2: For clarity: this moves the note that everyone is encouraged to contribute to the top (instead of making it a task bullet point). It also adds periods.

Usually teams need at least 1 hour to work through all the steps above. You can think of this as building a strong foundation for the work that happens throughout the rest of the week. 

```{warning}
It's common for teams to dive into the project work too early. Do you know the names of every person in your group? Have you heard from everyone about their learning goals and ideas for the project tasks? If you answer "no" to any of these, we recommend you take a bit more time working through the activities above. It will pay off for you later in the week!
```

* As progress is made, project outputs (code, notebooks, figures, documentation, etc.) are recorded in the team's GitHub repository (a good place for team members to get started is by making their own folder under the *contributors* directory).
* All team members check back in at the end of the day, review progress towards tasks, anything preventing progress, and planned next steps
* Project leaders may also want to share with the whole hackweek group: How the project is coming along, what are the objectives and tasks identified by the team today, and what other skills might your team need to be successful?

> ## Daily Team Standups

Reviewer 2: ```suggestion
## Days 1-5: Standups
```

> A "stand up" meeting model may be useful for your team to check in at the beginning and end of each day of the hackweek. During these short meetings, each team member is given a minute or so to briefly share 1) progress they have made on their tasks, 2) anything preventing progress, and 3) what they plan on working on next. 

Reviewer 2: ```suggestion
A "standup" meeting model may be useful for your team to check in at the beginning and end of each day of the hackweek. During these short meetings, each team member is given a minute or so to briefly share 1) progress they have made on their tasks, 2) anything preventing progress, and 3) what they plan on working on next.

The goal of these "standups" is to make sure everyone has the resources and support they need to achieve their hackweek goals - it can be hard to ask for help when you're working with a new team!
```

@aaarendt I'm sure you can improve the wording on this, but I'm going for reassurance that standups are meant to catch potential pain points early and not leave anyone behind rather than as a tool for measuring if people are accomplishing "enough".
